src/main/scala/de/upb/cs/uc4/hyperledger/connections/traits/ConnectionCourseTrait.scala

```diff
@@ -0,0 +1,59 @@
+package de.upb.cs.uc4.hyperledger.connections.traits
+
+import de.upb.cs.uc4.hyperledger.exceptions.traits.{HyperledgerExceptionTrait, TransactionExceptionTrait}
+
+trait ConnectionCourseTrait extends AbstractConnectionTrait {
+
+  /**
+   * Executes the "addCourse" query.
+   *
+   * @param jSonCourse Information about the course to add.
+   * @throws Exception if chaincode throws an exception.
+   * @return Success_state
+   */
+  @throws[TransactionExceptionTrait]
+  @throws[HyperledgerExceptionTrait]
+  def addCourse(jSonCourse: String): String
+  /**
```

Comment:
Use the same amount of new lines between methods and at least one

Reply:
nächster push